state/execution_test.go

```diff
@@ -84,6 +83,10 @@ func TestCreateProposalBlock_MempoolRemoveTxs(t *testing.T) {
 		proposerAddr,
 	)
 	assert.EqualValues(t, 1, mempool.TxsBytes())
+
+	tx, err := mempool.GetNextTxBytes(2, 1, nil)
+	assert.NoError(t, err)
+	assert.EqualValues(t, []byte{0x03}, tx)
```

Comment:
and then `mempool.GetNextTxBytes(2, 1, tx)` should return nil because all txs have been iterated